src/Catalyst.Abstractions/P2P/IPeerSettings.cs

```diff
@@ -30,9 +30,9 @@ namespace Catalyst.Abstractions.P2P
 {
     public interface IPeerSettings
     {
+        IPAddress PublicIpAddress { get; }	
```

Comment:
I don't understand, didn't you ask me to remove it yesterday?

Reply:
yes I realised yesterday, Was PublicKey.

Specifically @Xela101 had this PR https://github.com/catalyst-network/Catalyst.Framework/pull/1067

Which removed IKeySigner being injected to IPeerSettings.

We initially injected IKeySigner there to the settings can get the public key from the keystore loaded and not the json configs, which where causing clashes if people had the wrong settings in. For some reason this started to fail on all platforms except windows  on the CI only.

Locally for me these tests pass